src/Init/Data/Nat/Basic.lean

```diff
@@ -740,6 +809,9 @@ theorem not_eq_zero_of_lt (h : b < a) : a ≠ 0 := by
 theorem pred_lt' {n m : Nat} (h : m < n) : pred n < n :=
   pred_lt (not_eq_zero_of_lt h)
 
+theorem sub_one_lt' {n m : Nat} (h : m < n) : n - 1 < n :=
```

Comment:
Perhaps should be called `sub_one_lt_of_lt`. Prefer to only use primes for close variants of the unprimed theorem, not just to resolve a name conflict.

Reply:
Sounds good to me, I can do the name change to the `pred_lt'` as well.